examples/flask_example.py

```diff
@@ -0,0 +1,15 @@
+from flask import Flask
```

Comment:
Maybe add a docstring at the top of the file explaining how to use it. It's very basic but might be useful to people. Something about running this and then going to `http://127.0.0.1:5000/` in a browser.

Reply:
Can do!

Reply:
Added :D